src/utils/index.ts

```diff
@@ -46,6 +46,9 @@ export function throwOnErrorResponse(msg: Message) {
 }
 
 export function makeRandomBytes(size = 32) {
+  if (typeof globalThis.crypto === "undefined") {
+    return nodeCrypto.randomBytes(size)
+  }
```

Comment:
We weren't actually calling this anywhere in tests and it was failing because the WebCrypto API is DOM only and we are using (probably correctly) the Node environment. I added a little polyfill in case we're not in a browser.